modules/ROOT/pages/getting-started-proxy.adoc

```diff
@@ -1,89 +1,96 @@
+. Log in to Anypoint Platform and select *API Manager*.
 +
-. In the next screen, you have to complete the following fields:
+The API Administration window is displayed.
+. Select *Manage API > Manage API from Exchange* and in the Manage API from Exchange window, specify the following details:
 +
-image::getting-started-proxy-editing-api.png[editing-api,width=75%]
+- *API Name*: Type the first characters of the API name that you created in Exchange.
 +
-- *API Name*: Type the first characters of the API name (The same name you chose when you created your asset in Exchange), and
-then API Manager displays APIs to choose from. Select the API one that was recently created in Exchange.
+The list displays the full name of the API asset that you created in the previous procedure.
 +
-- *Asset Type*: For this example we are going to use HTTP, but you can also choose to create a RAML/OAS or WSDL API.
+- *Asset Type*: For this example, use HTTP, but you can also choose to create a RAML/OAS or WSDL API.
 +
 - *API Version*: Available versions in Exchange.
 +
-- *Managing Type*: You may choose between `Basic Endpoint` or `Endpoint with Proxy`. Here we are using the endpoint with proxy.
+- *Managing Type*: To deploy an API with proxy on CloudHUb, select *endpoint with proxy*.
++
+If you do not want to configure your API with proxy, select *Basic endpoint*. 
++
+- *Mule version:* If you are managing an API for Mule 4 or later, select this checkbox. 
 +
-- *Implementation URI*: Enter the URL where the API implementation resides. For this example we are using http://my-implementation.company.com
+In this example, create the API for Mule 4.
 +
-If you are managing an API for Mule 4 or above, please make sure you select the checkbox. In this example, we are creating the API for Mule 4.
+- *Implementation URI*: Enter the URL for where the API implementation resides. 
++
+For this example, use http://my-implementation.company.com:
++
+image::getting-started-proxy-editing-api.png[editing-api,width=75%]
 +
-----
-If you need further configuration, you may want to click in Advanced Options, and fill in the options available.
-----
+For additional configuration options, click *Advanced options*, and complete the details.
 +
-. Click on `Save`.
+. Click *Save*.
 
-After that, the API settings view is shown with the current information of the recently imported API.
+The API settings view is displayed with the details of the imported API. Your API instance is now created:
 
 image::getting-started-proxy-api-settings.png[created-api-in-admin-dashboard]
 
-So far, we have created an instance of an API Specification that can now be managed.
+[[configure_and_deploy_to_cloudhub]]
+== Configure a Proxy and Deploy to CloudHub
 
-== Configure and Deploy a Proxy to Cloudhub
+Deploying an API proxy to CloudHub enables you to protect your API implementation without having to worry about the infrastructure details, such as firewalls and so on. 
```

Comment:
Deploying an API proxy to CloudHub is the fastest way to protect your API..

Reply:
Done! Thanks!